libraries/AP_RTPS/AHRS.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2016 Intel Corporation. All rights reserved.
+ *
+ * Based on a file auto-generated by the fastcdrgen tool with the following
+ * Copyright and license:
+ * Copyright (C) 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <array>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+namespace eprosima {
+namespace fastcdr {
+    class Cdr;
+}
+}
+
+class AHRS {
```

Comment:
Would it be better if it were under a `rtps` namespace instead of changing the name?